{"id":"node_modules/core-js/library/modules/_iter-call.js","dependencies":[{"name":"/Users/karkran/Documents/nikhil personal/REACT-SECURITY/React-security/authentication/package.json","includedInParent":true,"mtime":1539692776000},{"name":"/Users/karkran/Documents/nikhil personal/REACT-SECURITY/React-security/authentication/.babelrc","includedInParent":true,"mtime":1539682095000},{"name":"/Users/karkran/Documents/nikhil personal/REACT-SECURITY/React-security/authentication/node_modules/core-js/package.json","includedInParent":true,"mtime":1539682157000},{"name":"./_an-object","loc":{"line":2,"column":23},"parent":"/Users/karkran/Documents/nikhil personal/REACT-SECURITY/React-security/authentication/node_modules/core-js/library/modules/_iter-call.js","resolved":"/Users/karkran/Documents/nikhil personal/REACT-SECURITY/React-security/authentication/node_modules/core-js/library/modules/_an-object.js"}],"generated":{"js":"// call something on iterator step with safe closing on error\nvar anObject = require('./_an-object');\nmodule.exports = function (iterator, fn, value, entries) {\n  try {\n    return entries ? fn(anObject(value)[0], value[1]) : fn(value);\n  // 7.4.6 IteratorClose(iterator, completion)\n  } catch (e) {\n    var ret = iterator['return'];\n    if (ret !== undefined) anObject(ret.call(iterator));\n    throw e;\n  }\n};\n","map":{"mappings":[{"source":"node_modules/core-js/library/modules/_iter-call.js","original":{"line":1,"column":0},"generated":{"line":1,"column":0}},{"source":"node_modules/core-js/library/modules/_iter-call.js","original":{"line":2,"column":0},"generated":{"line":2,"column":0}},{"source":"node_modules/core-js/library/modules/_iter-call.js","original":{"line":3,"column":0},"generated":{"line":3,"column":0}},{"source":"node_modules/core-js/library/modules/_iter-call.js","original":{"line":4,"column":0},"generated":{"line":4,"column":0}},{"source":"node_modules/core-js/library/modules/_iter-call.js","original":{"line":5,"column":0},"generated":{"line":5,"column":0}},{"source":"node_modules/core-js/library/modules/_iter-call.js","original":{"line":6,"column":0},"generated":{"line":6,"column":0}},{"source":"node_modules/core-js/library/modules/_iter-call.js","original":{"line":7,"column":0},"generated":{"line":7,"column":0}},{"source":"node_modules/core-js/library/modules/_iter-call.js","original":{"line":8,"column":0},"generated":{"line":8,"column":0}},{"source":"node_modules/core-js/library/modules/_iter-call.js","original":{"line":9,"column":0},"generated":{"line":9,"column":0}},{"source":"node_modules/core-js/library/modules/_iter-call.js","original":{"line":10,"column":0},"generated":{"line":10,"column":0}},{"source":"node_modules/core-js/library/modules/_iter-call.js","original":{"line":11,"column":0},"generated":{"line":11,"column":0}},{"source":"node_modules/core-js/library/modules/_iter-call.js","original":{"line":12,"column":0},"generated":{"line":12,"column":0}},{"source":"node_modules/core-js/library/modules/_iter-call.js","original":{"line":13,"column":0},"generated":{"line":13,"column":0}}],"sources":{"node_modules/core-js/library/modules/_iter-call.js":"// call something on iterator step with safe closing on error\nvar anObject = require('./_an-object');\nmodule.exports = function (iterator, fn, value, entries) {\n  try {\n    return entries ? fn(anObject(value)[0], value[1]) : fn(value);\n  // 7.4.6 IteratorClose(iterator, completion)\n  } catch (e) {\n    var ret = iterator['return'];\n    if (ret !== undefined) anObject(ret.call(iterator));\n    throw e;\n  }\n};\n"},"lineCount":13}},"hash":"88cf91ab6c242e08d06e8a56a5646c94","cacheData":{"env":{}}}